card/mod/04_settings/set/right/create.rb

```diff
@@ -1,3 +1,3 @@
-include Comment
+include Abstract::Permission
```

Comment:
...might actually be less confusing, since there's already a <modname>/lib dir....